store/tikv/lock_resolver.go

```diff
@@ -229,6 +229,9 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 		if err != nil {
 			return false, err
 		}
+		if l.LockType != kvrpcpb.Op_PessimisticLock && status.ttl == 0 {
+			lr.saveResolved(l.TxnID, status)
```

Comment:
Can we combine `TxnID` and Primary's `LockForUpdateTS` together as the key of `LockResolver.resolved` as a cache. I'm worried about the performance impact of removing the cache.

Reply:
The impact now is resolving pessimistic locks whose primary lock is prewrite lock type, will need more `check_txn_status` calls, we could make the returnd status certain or not in the future to lower the risk.